rpc/write.go

```diff
@@ -76,3 +78,32 @@ func encodeProgram(content []byte) (string, error) {
 	program := base64.StdEncoding.EncodeToString(buf.Bytes())
 	return program, nil
 }
+
+func guessABI(abis []interface{}) (*types.ABI, error) {
```

Comment:
works for now but we should ideally move this to the `Unmarshal` implementation for ABI